packages/editor-ui/src/views/VariablesView.vue

```diff
@@ -290,6 +291,18 @@ onBeforeUnmount(() => {
 				buttonType="secondary"
 				@click="goToUpgrade"
 			/>
+			<n8n-action-box
+				v-if="!canCreateVariables"
```

Comment:
```suggestion
				v-else-if="!canCreateVariables"
```

To make sure we never show both action boxes.

Reply:
Good catch, thanks, fixed and tested